docs/development/local-development.md

```diff
@@ -77,35 +78,26 @@ VS Code [integrated terminal](https://code.visualstudio.com/docs/editor/integrat
 
 #### Fork and Clone
 
-If you will contribute to the project, you should first "fork" the main project using the GitHub Website and then clone your fork locally.
+If you want to contribute to the project, you should first "fork" the main project using the GitHub Website and then clone your fork locally.
+The Renovate project uses the [Yarn](https://github.com/yarnpkg/yarn) package management system instead of npm.
 
-#### Install dependencies
+To ensure everything is working properly on your end, you must:
 
-We use [Yarn](https://github.com/yarnpkg/yarn) so run `yarn install` to install dependencies instead of `npm install`.
+1. Install all dependencies with `yarn install`.
+1. Make a build with `yarn build`, which should pass with no errors.
+1. Verify all tests pass and have 100% test coverage, by running `yarn test`.
+1. Verify the installation by running `yarn start`. You must see this error: `Fatal error: No authentication found for platform https://api.github.com/ (github)`
```

Comment:
This numbering works 👀

Reply:
> This numbering works :eyes:

I'm not sure what you mean here? Do you mean I'm not breaking anything?

Or are you surprised that this kind of numbering scheme works in Markdown?

```markdown
1. ordered list item 1.
1. ordered list item 2.
1. ordered list item 3.
```

Markdown just wants you to use a number for a ordered list, if you use `1.`'s it will generate the right numbering sequence.
This way you can re-arrange the ordered list without having to change the numbering scheme.

You can use 1. 2. 3. though, that also works, but it's more work when you want to edit the order later.

Reply:
Just was a little suprised. New stuff learned 😅